scriptmodules/ports/jumpnbump.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="jumpnbump"
+rp_module_desc="Jump 'n Bump, play cute bunnies jumping on each other's heads - Modernization fork"
+rp_module_help="Copy custom game levels (.dat) to $romdir/ports/jumpnbump"
+rp_module_licence="GPL2 https://gitlab.com/LibreGames/jumpnbump/raw/master/COPYING"
+rp_module_section="exp"
+rp_module_flags=""
+
+function depends_jumpnbump() {
+    getDepends libsdl2-dev libsdl2-net-dev libsdl2-mixer-dev libbz2-dev zlib1g-dev
+}
+
+function sources_jumpnbump() {
+    gitPullOrClone "$md_build" https://gitlab.com/LibreGames/jumpnbump.git
+}
+
+function build_jumpnbump() {
+    make clean
+    CFLAGS="$CFLAGS -fsigned-char" make PREFIX="$md_inst"
+}
+
+function install_jumpnbump() {
+    make PREFIX="$md_inst" install
+    strip "$md_inst"/bin/{gobpack,jnbpack,jnbunpack,jumpnbump}
```

Comment:
Please can you strip the binaries in the build stage?

Reply:
As I commented in the moonlight PR, stripping binaries should be done at the install phase as a good practice. However, I agree that for this particular module it won't affect doing it in the build phase as you request. If you still want to move it there and I will do it.

Reply:
I'm ok then with leaving it.